js/index.js

```diff
@@ -0,0 +1,166 @@
+const handleButtonClickNavigation = (url)=>{
+   window.location.href= url;
+}
+
+const toggleDropdown =(element)=>{
+    const targetDropdown = document.getElementById(element);
+
+    const allDropDowns = document.querySelectorAll(".dropdown-content");
+
+   allDropDowns.forEach(dropdown =>{
+    if(dropdown.id !== element && dropdown.classList.contains("show")){
+        dropdown.classList.remove("show")
+    }
+   })
+   if(targetDropdown){
```

Comment:
I suggest formatting this file as well (spaces after ifs, before brackets, extra lines etc.). Perhaps look at prettier / eslint extensions for vscode